### PR TITLE
add ember-django-adapter as in-repo addon

### PR DESCRIPTION
**Pull request by future-pirate-king**

### CODE REVIEW

File: tests/unit/serializers/drf-test.js [hunk 1]
```diff
@@ -1,14 +1,82 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
+// import sinon from 'sinon';
```
Comment (Yibaebi) [inline tests/unit/serializers/drf-test.js:3]: Can we take this off if it's not required?

File: tests/unit/serializers/drf-test.js [hunk 2]
```diff
@@ -1,14 +1,82 @@
...
 
-  // Replace this with your real tests.
-  test('it exists', function (assert) {
-    let store = this.owner.lookup('service:store');
-    let serializer = store.serializerFor('drf');
+  test.skip('serializeIntoHash', function (assert) {
+    let serializer = this.owner.lookup('serializer:application');
 
-    assert.ok(serializer);
+    // serializer.serialize = sinon.stub().returns({ serialized: 'record' });
```
Comment (Yibaebi) [inline tests/unit/serializers/drf-test.js:11]: This also?

File: tests/unit/serializers/drf-test.js (hunk 1 above)
Comment (future-pirate-king) [reply tests/unit/serializers/drf-test.js:3]: no it is for reference for skipped test

File: tests/unit/serializers/drf-test.js (hunk 2 above)
Comment (future-pirate-king) [reply tests/unit/serializers/drf-test.js:11]: no it is for reference for skipped test

**Review by Yibaebi — APPROVED** [review]

**Pull request merged**